src/main/java/edu/ucsb/cs156/gauchoride/entities/Ride.java

```diff
@@ -0,0 +1,38 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+// import io.swagger.annotations.ApiModelProperty;
+
+import javax.persistence.GeneratedValue;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "ride")
+public class Ride {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private String student;
+
+//   @ApiModelProperty(allowableValues = "Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday")
+  private String day;
+  
+  private String start;
+  private String end;
```

Comment:
It would be a good idea to make the format of start and end obvious from the names, e.g. startHHMM endHHMM or a comment such as:
```
  private String start; // format: HH:MM e.g. "13:45" 
```

or:

```
private String start; // format: HH:MMAM e.g. "02:30PM" or "11:30AM"
```